src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -39,6 +39,7 @@ import styles from './UniversalNavbarExpanded.module.scss'
  * @param {string} logoHref - Href for the logo
  * @param {object} links - URLs and text
  * @param {boolean} estimateExperiment - enable the estimate experiment button/copy
+ * @param {string} singleCta - A single CTA URL to link to in a reduced version of the navbar
```

Comment:
Update to match the propTypes def of `singleCta = { href: string, title: string }`